packages/yourloops/layout/caregiver-layout.tsx

```diff
@@ -39,15 +39,21 @@ export function CaregiverLayout(): JSX.Element {
   return (
     <PatientProvider>
       <DashboardLayout>
-        <Switch>
-          <Route exact path="/not-found" component={InvalidRoute} />
-          <Route exact path="/preferences" component={ProfilePage} />
-          <Route exact path="/notifications" component={NotificationsPage} />
-          <Route path="/patient/:patientId" component={PatientDataPage} />
-          <Route exact path="/home" component={HomePage} />
-          <Redirect exact from="/" to="/home" />
-          <Redirect to="/not-found" />
-        </Switch>
+        <Routes>
+          <Route path="/not-found" element={<InvalidRoute />} />
+          <Route path="/preferences" element={<ProfilePage />} />
+          <Route path="/notifications" element={<NotificationsPage />} />
+          <Route path="/patient/:patientId/*" element={<PatientDataPage />} />
+          <Route path="/home" element={<HomePage />} />
+          <Route
+            path="/"
+            element={<Navigate to="/home" replace />}
+          />
+          <Route
+            path="*"
+            element={<Navigate to="/not-found" replace />}
+          />
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Why not on one line like the others above ?